src/views/router_views/individualPscList/individualPscList.njk

```diff
@@ -0,0 +1,43 @@
+{% extends "layouts/default.njk" %}
+
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+
+{% set title = i18n.individual_psc_list_main_title %}
+{% set dummyRadioButtons = i18n.dummyRadioButtons %}
+{% set dummyPscOption = i18n.dummyPscOption %}
```

Comment:
You don't need to create a variable because you can use the localisation i18n.dummyPscOption directly in the code.